src/session.ts

```diff
@@ -179,12 +179,17 @@ export class SessionManager implements Middleware {
             }
         }
 
+        const startupBanner = this.sessionSettings.integratedConsole.suppressStartupBanner
+            ? ""
+            : "`n`n                  =====> PowerShell Integrated Console <=====`n`n";
```

Comment:
Isn't this the default? We should just let PSES have this string, no?

Reply:
I added it to be explicit in case we want to change it in future. On the other hand the duplication isn't ideal and I really don't like needing to write a PowerShell string literal in JS.

Hmmm, ok I'm convinced we should leave it as a the default now.

If we wanted to make it something like `==> VSCode PowerShell Extension <==` later, I'd imagine us doing it in the client. But for now I think I agree that leaving it unset in the default case is best.